HenBot/TagExistenceChecker.cs

```diff
@@ -18,7 +18,10 @@ private static async Task<bool> CheckIfTagQueryExists(string tagQuery)
         var tags = tagQuery.Split(' ');
         foreach (var tag in tags)
         {
-            var tagRes = await GelbooruSourceService.GetTagAsync(tag);
+            if(tag.Contains("user:"))
+                continue;
+            var tempTag = tag.Replace("~", "").Replace("{", "").Replace("}", "").Replace("*", "%").Replace(":general", "").Replace(":sensitive", "").Replace(":explicit", "").Replace("questionable", "");
```

Comment:
```suggestion
            var pattern = @"[~{}*:]|:general|:sensitive|:explicit|questionable";
            var tempTag = Regex.Replace(tag, pattern, "");
```

Idk weather you use word wraps in your IDE, or you're just a masochist... Anyway, it will be better and will not be generating lots of strings during runtime

FYI: https://regex101.com/ - very useful